### PR TITLE
Adds a ckey whitelist, stripping out some dead code

### PR DESCRIPTION
**Pull request by optimumtact**
Old job whitelist that wasn't used has been removed.

Config has been repurposed, uncomment USEWHITELIST, only ckey's in
admins.txt or whitelist.txt in the config will be allowed to connect to
the server

Reason for change: https://tgstation13.org/phpBB/viewtopic.php?f=2&t=8691#p231560

Also I suppose private servers may find it useful

### CODE REVIEW

**Comment by kevinz000** [comment]
add me to the whitelist :^)

**Review by Iamgoofball — APPROVED** [review]

File: config/whitelist.txt [hunk 1]
```diff
@@ -0,0 +1,7 @@
+###############################################################################################
+# Basically, a list of ckeys who make it past user connection                                 #
+# Case is not important for ckey.                                                             #
+###############################################################################################
+#Optimumtact
+#kevinz000
+#Iamgoofball
```
Comment (PKPenguin321) [inline config/whitelist.txt]: Put me in coach

**Comment by optimumtact** [comment]
put yourself in when it's merged

**Comment by ExcessiveUseOfCobblestone** [comment]
put rshoe in

**Comment by PKPenguin321** [comment]
Tfw you get cucked by rshoe

**Comment by PKPenguin321** [comment]
close enough!

**Comment by XDTM** [comment]
put me in the screenshot

**Comment by optimumtact** [comment]
@tgstation/commit-access merge and squash pls

**Comment by PKPenguin321** [comment]
no mergebegging!

**Pull request merged**